docs/mindsdb-docs/docs/sql/tutorials/customer-churn.md

```diff
@@ -21,18 +21,11 @@ The first step will be to log into [Mindsdb Cloud](https://docs.mindsdb.com/depl
 
 ### Upload a data file to MindsDB Cloud
 
-The datafile can be uploaded directly to Mindsdb Cloud for you to query. 
+In this tutorial, we use the customer churn data file. You can download it [here](https://github.com/mindsdb/mindsdb-examples/blob/master/classics/customer_churn/raw_data/WA_Fn-UseC_-Telco-Customer-Churn.csv).
```

Comment:
Can we rename the data file to a dataset to keep the consistency?

Reply:
Yes.